superset-frontend/src/components/Select/utils.tsx

```diff
@@ -218,7 +218,7 @@ export const mapOptions = (values: SelectOptionsType) =>
   values.map(opt => ({
     children: opt.label,
     key: opt.value,
-    value: opt.value,
     label: opt.label,
     disabled: opt.disabled,
+    ...opt,
```

Comment:
Do you intend `options` can override the children over `opt.label`? 

You can also remove the above exact matching(label, disabled) with this spreading.

```suggestion
    ...opt,
```